code/modules/mapping/map_template.dm

```diff
@@ -26,8 +31,10 @@
 			cached_map = parsed
 	return bounds
 
-/datum/parsed_map/proc/initTemplateBounds(datum/map_template/template)
-
+/datum/map_template/proc/initTemplateBounds(list/bounds)
+	if (!bounds) //something went wrong
+		message_admins("[name] template failed to initialize!")
+		return
```

Comment:
Add a `stack_trace()`?
Messaging admins might be lost.

Reply:
now a stack_trace()